src/session10/homework/bank_account_management/BankAccount.java

```diff
@@ -0,0 +1,48 @@
+package session10.homework.bank_account_management;
+
+import java.util.UUID;
+
+public class BankAccount {
+
+    String accountHolderName;
+    UUID accountNumber;
+    double balance;
+
+    public BankAccount(String accountHolderName) {
+        this.accountHolderName = accountHolderName;
+        this.accountNumber = UUID.randomUUID();
+        this.balance = 0;
+    }
+
+    public double getBalance() {
+        return balance;
+    }
+
+    public String getAccountDetails() {
+        return "Account holder name: " + accountHolderName + "\n" +
+                "Account number: " + accountNumber;
+    }
+
+    public void deposit(double amount) {
+        if (amount > 0) {
+            this.balance += amount;
+        }
+    }
+
+    public void withdraw(double amount) {
+        if (amount > 0) {
```

Comment:
We use 0 in multiple places in this class. Can we consider having a constant with this value?